python/extras/util.py

```diff
@@ -8,10 +9,13 @@
 from dataclasses import dataclass
 from functools import wraps
 from pathlib import Path
-from typing import Callable, List, Optional, Sequence, Tuple, Union
+from typing import Callable, List, Sequence, Tuple, get_args, TypeVar
 
+import tensorflow as tf
 import numpy as np
 
+bfloat16 = tf.bfloat16.as_numpy_dtype
+
```

Comment:
TensorFlow is a really big library for this very targeted purpose, so if I install with something like:
```
pip install tensorflow-cpu
```

Also, when I load a module with these imports, I get output like below (which I could probably figure out how to silence, but is currently irritating):

```
2024-09-25 08:36:50.658114: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-09-25 08:36:50.686614: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI AVX512_BF16 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
```